### PR TITLE
Add /labs route with layout distinct from GCN

### LINKED ISSUE #1582: Refactor GCN content and root routes into subfolder

### PR DESCRIPTION
**Pull request by lpsinger**
This is an alternative implementation of #1619.

# TL;DR

The following two tables illustrates the difference in file naming between #1619 and this patch.

#1619:

| Source file path | URL |
| - | - |
| `app/route/_gcn+/foo.bar.tsx` | `https://gcn.nasa.gov/foo/bar` |
| `app/route/labs+/bat.baz.tsx` | `https://gcn.nasa.gov/labs/bat/baz` |

This PR:

| Source file path | URL |
| - | - |
| `app/route/_gcn.foo.bar.tsx` | `https://gcn.nasa.gov/foo/bar` |
| `app/route/labs.bat.baz.tsx` | `https://gcn.nasa.gov/labs/bat/baz` |

# Rationale

Remix projects use a [flat route file naming convention](https://remix.run/docs/en/main/file-conventions/routes) such that a source file with the name `app/routes/foo.bar.bat.tsx` or `app/routes/foo.bar.bat.md` maps to the URL `/foo/bar/bat`. This naming convention places all of the sources files for all of the URLs in a single folder.

In #1581, we had installed the third-party package [remix-flat-routes](https://github.com/kiliman/remix-flat-routes) which allows you to use folders in your source tree to organize your routes. We were planning to put all of the GCN routes in the `app/routes/_gcn+` directory (mapping to URLs starting with `/`) and all of the ACROSS routes in `app/routes/labs+` (mapping to URLs starting with `/labs`). However, as I was working on that refactoring in #1619, I came across a bug in remix-flat-routes that makes it incompatible with Remix's own route convention (https://github.com/kiliman/remix-flat-routes/pull/83). I haven't found a workaround for this bug.

Although remix-flat-routes' mapping of filenames to URLs is similar to the core Remix routing convention, it is incompatible in a few subtle ways that would break the existing GCN code. I think it will be safer in the long run to use Remix's core route convention, avoiding the risk of other divergences and incompatibilities.

Reverts #1581.

Fixes #1582.

### CODE REVIEW

**Review by dakota002 — APPROVED** [review]

**Comment by jak574** [comment]
I pulled these changes and tested with the ACROSS API code, and everything works fine.

**Review by jak574 — APPROVED** [review]
